docs/src/index.md

```diff
@@ -99,33 +118,40 @@ The particle filter parameters are saved in the following data structure:
 ParticleDA.FilterParameters
 ```
 
-## Example: tsunami modelling with the bootstrap filter
+## Example: estimating the state in a tsunami model
 
-A full example of a model interfacing `ParticleDA` is available in
-`test/model/model.jl`.  This model represents a tsunami and is partly based on
-the [tsunami data assimilation code](https://github.com/tktmyd/tdac) by Takuto
-Maeda.  You can run it with
+A full example of a model interfacing `ParticleDA` is available in `test/models/llw2d.jl`. 
+This model represents a simple two dimensional simulation of tsunami dynamics and is 
+partly based on the [tsunami data assimilation code](https://github.com/tktmyd/tdac) by 
+Takuto Maeda. The particle filter can be run with observations simulated from the model
+as follows
 
 ```julia
 # Load ParticleDA
 using ParticleDA
 
 # Save some variables for later use
 test_dir = joinpath(dirname(pathof(ParticleDA)), "..", "test")
-module_src = joinpath(test_dir, "model", "model.jl")
+llw2d_src = joinpath(test_dir, "models", "llw2d.jl")
 input_file = joinpath(test_dir, "integration_test_1.yaml")
+observation_file = tempname()
 
 # Instantiate the test environment
 using Pkg
 Pkg.activate(test_dir)
 Pkg.instantiate()
 
 # Include the sample model source code and load it
-include(module_src)
-using .Model
+include(llw2d_src)
+using .LLW2d
+
+# Simulate observations from the model to use 
+simulate_observations_from_model(LLW2d.init, input_file, observation_file)
 
-# Run the particle filter using the `init` file defined in the `Model` module.
-run_particle_filter(Model.init, input_file, BootstrapFilter())
+# Run the (optimal proposal) particle filter with simulated observations
+final_states, final_statistics = run_particle_filter(
+  LLW2d.init, input_file, observation_file, OptimalFilter
+)
```

Comment:
Should state that if one is running on non-Intel chip you should select ` ParticleDA.NaiveMeanSummaryStat`. Additionally add a subsection stating this. Please see the suggested text.

Reply:
Now added comment to indicate this.